ocs_ci/resiliency/network_faults.py

```diff
@@ -80,7 +80,7 @@ def _get_all_node_network_interfaces(self, interface_types):
                         iface.strip() for iface in output.splitlines() if iface.strip()
                     ]
                     interfaces.extend(ifaces)
-                except CommandFailed as e:
+                except (CommandFailed, subprocess.TimeoutExpired) as e:
```

Comment:
You catch subprocess.TimeoutExpired but only log and continue; consider a retry/back‑off strategy if transient or a cleanup function in failure case, to make sure that network recovered